test/functionalTest/harnessFunctions.sh

```diff
@@ -1242,13 +1246,17 @@ function coapCurl()
     then
       if [ "$_outFormat" == application/xml ] || [ "$_outFormat" == "" ]
```

Comment:
Because 0000_deprecated_checkings/xml_support.test needs to attemp use XML. Maybe the same could be done in a simpler way, but not in the scope of this PR (that's why I have leave the FIXME P10 here).